cmd/release-controller/http_helper_test.go

```diff
@@ -196,7 +196,7 @@ func Test_calculateReleaseUpgrades(t *testing.T) {
 				}
 			}
 			if got := calculateReleaseUpgrades(tt.release, tt.tags, tt.graph(), false); !reflect.DeepEqual(got, tt.want) {
-				t.Errorf("%s", diff.ObjectReflectDiff(tt.want, got))
+				t.Errorf("%s", cmp.Diff(tt.want, got))
```

Comment:
We prefer to keep using diff.ObjectReflectDiff in general - we reserve the right to change implementation later (so better to leave this one).